commit fb43d79e12ce132fc127cc0481ff5a6bdbcd1afe
Author: Miklos Vajna <vmiklos@collabora.co.uk>
Date:   Tue Nov 25 11:26:34 2014 +0100

    Od[gp]Generator: declare meta namespace
    
    Other generators had it already, and without this, it's not possible to
    declare creation date.

diff --git src/OdgGenerator.cxx src/OdgGenerator.cxx
index 6a2b1a4..ca4d4f0 100644
--- src/OdgGenerator.cxx
+++ src/OdgGenerator.cxx
@@ -283,6 +283,7 @@ bool OdgGeneratorPrivate::writeTargetDocument(OdfDocumentHandler *pHandler, OdfS
 	docContentPropList.addAttribute("xmlns:draw", "urn:oasis:names:tc:opendocument:xmlns:drawing:1.0");
 	docContentPropList.addAttribute("xmlns:table", "urn:oasis:names:tc:opendocument:xmlns:table:1.0");
 	docContentPropList.addAttribute("xmlns:xlink", "http://www.w3.org/1999/xlink");
+	docContentPropList.addAttribute("xmlns:meta", "urn:oasis:names:tc:opendocument:xmlns:meta:1.0");
 	docContentPropList.addAttribute("xmlns:dc", "http://purl.org/dc/elements/1.1/");
 	docContentPropList.addAttribute("xmlns:svg", "urn:oasis:names:tc:opendocument:xmlns:svg-compatible:1.0");
 	docContentPropList.addAttribute("xmlns:fo", "urn:oasis:names:tc:opendocument:xmlns:xsl-fo-compatible:1.0");
diff --git a/src/OdpGenerator.cxx b/src/OdpGenerator.cxx
index 4345c40..8ba4dc0 100644
--- src/OdpGenerator.cxx
+++ src/OdpGenerator.cxx
@@ -444,6 +444,7 @@ bool OdpGeneratorPrivate::writeTargetDocument(OdfDocumentHandler *pHandler, OdfS
 	docContentPropList.addAttribute("xmlns:fo", "urn:oasis:names:tc:opendocument:xmlns:xsl-fo-compatible:1.0");
 	docContentPropList.addAttribute("xmlns:config", "urn:oasis:names:tc:opendocument:xmlns:config:1.0");
 	docContentPropList.addAttribute("xmlns:xlink", "http://www.w3.org/1999/xlink");
+	docContentPropList.addAttribute("xmlns:meta", "urn:oasis:names:tc:opendocument:xmlns:meta:1.0");
 	// WARNING: this is not ODF!
 	docContentPropList.addAttribute("xmlns:ooo", "http://openoffice.org/2004/office");
 	docContentPropList.addAttribute("xmlns:officeooo", "http://openoffice.org/2009/office");
